packages/core/src/Utils/Datadog/index.ts

```diff
@@ -41,5 +41,6 @@ datadogRum.init({
     version: dataDogVersion,
     trackFrustrations: true,
     enableExperimentalFeatures: ['clickmap'],
+    excludedActivityUrls: [/^https:\/\/api.telegram.org.*$/],
```

Comment:
this is the log that's coming from bot package.
please add it there instead of this one.